charts/kube-prometheus-stack/values.yaml

```diff
@@ -4503,6 +4503,10 @@ prometheus:
     ## in Prometheus so it may change in any upcoming release.
     allowOverlappingBlocks: false
 
+    ## Specifies the validation scheme for metric and label names.
+    ## Supported values are: Legacy, UTF8
+    nameValidationScheme: Legacy
```

Comment:
I would probably make this value an empty string by default though so that when the default behavior changes upstream of it does we follow up with the intent